adhocracy4/comments_async/templatetags/react_comments_async.py

```diff
@@ -27,16 +28,24 @@ def react_comments_async(context, obj):
     has_comment_permission = user.has_perm(permission, obj)
 
     would_have_comment_permission = NormalUser().would_have_perm(
-        permission, obj
-    )
+        permission, obj)
+
+    comments_contenttype = ContentType.objects.get_for_model(Comment)
+    pk = obj.pk
+
+    with_categories = bool(with_categories)
 
     comments_api_url = reverse('comments-list',
                                kwargs={'content_type': contenttype.pk,
                                        'object_pk': obj.pk}
                                )
 
-    comments_contenttype = ContentType.objects.get_for_model(Comment)
-    pk = obj.pk
+    comment_category_choices = getattr(settings, 'A4_COMMENT_CATEGORIES', None)
+    if comment_category_choices:
+        comment_category_choices = dict(
+            (x, str(y)) for x, y in comment_category_choices)
+    else:
+        raise ImproperlyConfigured('set A4_COMMENT_CATEGORIES in settings')
```

Comment:
Some projects might only use the comments without categories. With the error they also have to set them to do that. Öhm, maybe we can put in some default categories instead? 
@rmader Or do you have a better idea?

Reply:
The ones from a+ seem to be good default categories. Should we use them?
We need to be pretty careful if we already use debate comments and change the categories later, though. So, maybe not such a great idea after all.
Oh, we can only ask, set and raise when with_categories is True :woman_facepalming: 
I will put that in!

Reply:
I'm going to try and mirror to see if I understand :p
So if no comment_category_choices it will trigger the ImproperlyConfigured error. 
You suggest to solve this we have a default setting for categories so that the error is not triggered but this will cause issues when we have changeable categories.
Did I get that right?
Can we not just add another argument to the ifstatment with if categories = true and comment_category_choices: blah blah, or have I really got the wrong end of the stick?